### PR TITLE
Nt/refres fcm token

### PR DESCRIPTION
**Pull request by noumantahir**
### What does this PR?
Keeping local notification settings and remote settings plus token in sync by updating backend on startup

### CODE REVIEW

File: src/screens/application/container/applicationContainer.js [hunk 1]
```diff
@@ -854,7 +882,6 @@ export default connect(
     // Application
     isDarkTheme: state.application.isDarkTheme,
     selectedLanguage: state.application.language,
-    notificationSettings: state.application.isNotificationOpen,
```
Comment (feruzm) [inline src/screens/application/container/applicationContainer.js:857]: Settings and Login screens use this I believe, wouldn't that get broken if removed?

Comment (noumantahir) [reply src/screens/application/container/applicationContainer.js:857]: I doubt that, it is not being used in this reference

Comment (noumantahir) [reply src/screens/application/container/applicationContainer.js:857]: giving it a second look just in case

Comment (noumantahir) [reply src/screens/application/container/applicationContainer.js:857]: I am pretty confident it won't break anything, check the screenshot fro details
Screenshot 2021-05-29 at 12 51 23 PM

**Review by feruzm — APPROVED** [review]
great, lgtm 👍

**Pull request merged**